.graphqlrc.yml

```
@@ -0,0 +1,2 @@
+schema: './node_modules/@nft/api-graphql/schema.graphql'
+documents: '{components,pages}/**/*.{graphql,gql}'
```

Maybe also add hooks folder? As we have some graphql there as well.
```suggestion
documents: '{components,hooks,pages}/**/*.{graphql,gql}'
```